### PR TITLE
Add https configuration item for data path.

### PR DESCRIPTION
**Pull request by flei2000**
Config and handle http/https for data path.

### CODE REVIEW

File: ecstest/testbase.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def setUp(self):
         self.data_conn = S3Connection(
             aws_access_key_id=cfg['ACCESS_KEY'],
             aws_secret_access_key=cfg['ACCESS_SECRET'],
-            is_secure=False,
+            is_secure=cfg['ACCESS_SSL'],
```
Comment (MingatRubicon) [inline ecstest/testbase.py]: will this work? we need a string here or just a flag?

Comment (flei2000) [inline ecstest/testbase.py]: config.py convert everything to bool.

Comment (stevendgonzales) [inline ecstest/testbase.py]: Just note that converting the string "false" to a boil will evaluate as True.  All strings in Python evaluate as True except for an empty string ""

Comment (flei2000) [inline ecstest/testbase.py]: Yes. Just found that. What's your suggestion?

Comment (stevendgonzales) [inline ecstest/testbase.py]: In the other Configs there I converted to int before bool, so the user supplies a 0 or 1.  Alternatively you could write a function to convert that does comparison:

def convert_str_bool(val):
    return lower(val) == "true"

Comment (flei2000) [inline ecstest/testbase.py]: Could we first merge this https feature? Then we start another branch to unify shell var name and bool expression.

**Pull request merged**